conftest.py

```diff
@@ -0,0 +1,56 @@
+"""
+This file configures a local pytest plugin, which allows us to configure plugin hooks to control the
+execution of our tests. Either by loading in fixtures, configuring directories to ignore, etc
+
+Local plugins: https://docs.pytest.org/en/3.10.1/writing_plugins.html#local-conftest-plugins
+Hook reference: https://docs.pytest.org/en/3.10.1/reference.html#hook-reference
+"""
+import os
+import re
+import sys
+
+import pytest
+
+PY_DIR_PATTERN = re.compile(r'^py[23][0-9]$')
+
+
+# Determine if the folder should be ignored
+# https://docs.pytest.org/en/3.10.1/reference.html#_pytest.hookspec.pytest_ignore_collect
+# DEV: We can only ignore folders/modules, we cannot ignore individual files
+# TODO: Should this be `pytest_collect_directory`? or `pytest_collect_file`?
```

Comment:
Do we need this TODO? I think maybe directory makes more sense since we can't ignore individual files but not a big deal imo.

Reply:
We *might* be able to ignore individual files, just not sure if we want to or not.

Having as their own `py35/` directory means we can also selectively run `flake8` for Python3 files as well.